resources/views/home.blade.php

```diff
@@ -20,12 +20,12 @@
         RenderActivePlayersComponent();
     ?>
 
-    <x-user.online-count-chart />
+    <div class="mb-8">
+        <x-user.online-count-chart />
+    </div>
```

Comment:
The proximity of the "Sets in Progress" heading to the chart was driving me crazy, so I made a subtle tweak here.

**Before**
![Screenshot 2023-09-26 at 7 00 09 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/c60a7066-53c5-46a7-8117-18f930fafc9c)

**After**
![Screenshot 2023-09-26 at 6 59 58 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/3034652e-0a53-465a-979a-fe3bfaa622e1)